backend/packages/wps-shared/src/wps_shared/db/crud/auto_spatial_advisory.py

```diff
@@ -821,36 +844,28 @@ async def get_provincial_rollup(
     run_type: RunTypeEnum,
     run_datetime: datetime,
     for_date: date,
-    fuel_type_raster_id: int,
```

Comment:
Are we still able to determine which fuel raster was used in calculating the stats?